react-ui/src/components/Reviewer/Questions.js

```diff
@@ -10,39 +10,27 @@ class Questions extends Component {
   constructor(props) {
     super(props);
     this.handleSubmit = this.handleSubmit.bind(this);
-    this.updateAnswer = this.updateAnswer.bind(this);
-    this.handleChange = this.handleChange.bind(this);
+    this.handleInputChange = this.handleInputChange.bind(this);
     this.state = {
       answer: '',
       submitResponse: false,
       questionsLeft: 0, // TODO need some way to stop at the last question and go to the final page
     };
   }
 
-  updateAnswer(e) {
-    e.preventDefault;
-    const answer = this.refs.input ? this.refs.input.value : this.state.answer;
-    this.setState({
-      answer: answer,
-    })
-    this.handleSubmit(answer);
-  }
-
   //submit question and answer to reviews route
   handleSubmit(e) {
     e.preventDefault;
-    let answer = this.state.answer;
-    console.log(this.refs.input);
-    let question = find(Number(this.props.match.params.id) - 1).questionText;
-    let review_id; - // TODO where is the review_id? how can we access it here? componentDidMount?
-    console.log("answer", answer); // TODO this is coming back undefined no matter how I try to get it
-    console.log("question", question);
+
+    const answer = this.state.answer;
+    const question = find(Number(this.props.match.params.id) - 1).questionText;
+    const { reviewId } = this.props;
 
     // on question submits save review_id and question and answer to responses table
-    axios.patch(`/api/add-response/${review_id}`, {
-      // review_id: review_id,
+    axios.patch(`/api/add-response/${reviewId}`, {
+      review_id: reviewId,
       question: question,
-      answer: answer,
+      response: answer,
```

Comment:
Here is where we send over the params to the server and that endpoint was expecting "response" not "answer."